DataAccess.EF/Repositories/ApplicationRepository.cs

```diff
@@ -0,0 +1,49 @@
+﻿using DataAccess.EF.DataAccess;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccess.EF.Repositories;
+
+public class ApplicationRepository<TEntityType> : IApplicationRepository<TEntityType> where TEntityType : class
+{
+    public ApplicationRepository(ApiContext db)
+    {
+        Context = db;
+    }
+
+    public ApiContext Context { get; }
```

Comment:
I need to have an access to Items and Categories, can I use this  property to return a context, which contain these entities?

Reply:
I think I can do smth like this
https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/getting-started-with-ef-5-using-mvc-4/implementing-the-repository-and-unit-of-work-patterns-in-an-asp-net-mvc-application

Reply:
Yes, you can. But repository shall be in request scope for the context handling